docs/specs/servers/blockchain/blockchain-permissions-api.md

```diff
@@ -0,0 +1,174 @@
+# Blockchain API Sessions and Permissions
+
+## Sessions permissions storage
+
+### Get permissions list for account
+
+Used to get account list of active sessions
+
+`GET /v1/sessions/{address}?projectId={projectId}`
+
+* `address` - CAIP-10 address format.
+* `projectId` - Required. The project identifier.
+
+#### Success response body:
+
+```typescript
+{
+    pci: string[]
+}
+```
+
+* `pci` - List of sessions PCIs for this account.
+
+#### Response error codes:
+
+* `400 Bad request` - Wrong requested address format.
+
+### Get permission by PCI
+
+Used to get permission by PCI
+
+`GET /v1/sessions/{address}/{pci}?projectId={projectId}`
+
+* `address` - CAIP-10 address format.
+* `pci` - Permission identifier.
+* `projectId` - Required. The project identifier.
+
+#### Success response body:
+
+```typescript
+{
+    "permissionType": string,
+    "data": string,
+    "required": boolean,
+    "onChainValidated": boolean
+}
+```
+
+#### Response error codes:
+
+* `400 Bad request` - Wrong requested address format or PCI not found.
+
+## Add a new permission 
+
+Creating a new permission session for the account
+
+`POST /v1/sessions/{address}?projectId={projectId}`
+
+* `address` - CAIP-10 address format.
+* `projectId` - Required. The project identifier.
+
+### Request body:
+
+The POST request body should be in JSON format and following schema:
+
+```typescript
+{
+    permission:
+        {
+          "permissionType": string,
+          "data": string,
+          "required": boolean,
+          "onChainValidated": boolean
+        }
+}
+```
+
+#### Success response body:
+
+Response will contain a new generated key and PCI of the new permission.
+
+```typescript
+{
+    pci: string,
+    key: string
+}
+```
+
+* `pci` - New unique permission controller identifier.
+* `key` - Generated signing (private) ECDSA P256 key in DER, SEC1 format encoded by Base64.
```

Comment:
This should be `secp256k1` public key and better if we deal it in `did:key` format bc it would be un-necessary decoding/encoding the keys on client side